app/src/androidTest/java/org/wordpress/aztec/demo/BetterClickAction.java

```diff
@@ -0,0 +1,120 @@
+package org.wordpress.aztec.demo;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayingAtLeast;
+import static org.hamcrest.Matchers.allOf;
+
+import android.support.test.espresso.core.deps.guava.base.Optional;
+import android.support.test.espresso.PerformException;
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.support.test.espresso.action.CoordinatesProvider;
+import android.support.test.espresso.action.PrecisionDescriber;
+import android.support.test.espresso.action.Tap;
+import android.support.test.espresso.action.Tapper;
+import android.support.test.espresso.util.HumanReadables;
+import android.view.View;
+import android.view.ViewConfiguration;
+import android.webkit.WebView;
+import org.hamcrest.Matcher;
+
+/**
+ * Enables clicking on views with 65% or greater displaying.
+ */
+public final class BetterClickAction implements ViewAction {
+    private final CoordinatesProvider coordinatesProvider;
+    private final Tapper tapper;
+    private final PrecisionDescriber precisionDescriber;
+    private final Optional<ViewAction> rollbackAction;
+    public BetterClickAction(Tapper tapper, CoordinatesProvider coordinatesProvider,
+                              PrecisionDescriber precisionDescriber) {
+        this(tapper, coordinatesProvider, precisionDescriber, null);
+    }
+    public BetterClickAction(Tapper tapper, CoordinatesProvider coordinatesProvider,
+                              PrecisionDescriber precisionDescriber, ViewAction rollbackAction) {
+        this.coordinatesProvider = coordinatesProvider;
+        this.tapper = tapper;
+        this.precisionDescriber = precisionDescriber;
+        this.rollbackAction = Optional.fromNullable(rollbackAction);
+    }
+    @Override
+    @SuppressWarnings("unchecked")
+    public Matcher<View> getConstraints() {
+        Matcher<View> standardConstraint = isDisplayingAtLeast(65);
```

Comment:
Yes, the 65 here is a percentage: The default `ClickAction` in Espresso checks to make sure at least 90% of the element is displayed on-screen before clicking on it. The 65 here checks to make sure at least 65% of the element is displayed on-screen instead.